suniti heading font changes to ariel, h2 color changed to black.

sunitijain111 opened the pull request:
Closes #95 

hey! in public/ css-styles i made a couple of changes.
I have change heading fonts to ARIEL and h2 colour to black.

lisa761 commented on public/style.css:
```diff
@@ -60,11 +60,11 @@ a:focus{
 
 h2 {
-  color: #313338;  
+  color: #ffffff;  
```
`#ffffff` can also be written as `#fff`, it means the same thing. Plus `#fff` is actually white and not black.  But I don't see anything changing because of this?

sunitijain111 commented:
I changed font to comic-sans too and one more font. It was showing changes.
About colour, i changed to other colours, it wasn't affecting the website.
So i let it unchanged.

On Sun, 19 Apr 2020, 11:17 pm Lisa Chandra, <notifications@github.com>
wrote:

> *@lisa761* commented on this pull request.
>
> @sunitijain111 <https://github.com/sunitijain111>, hey good work!
> Although I am not too sure about Ariel instead of the current font. We can
> wait for what @soumyaa1804 <https://github.com/soumyaa1804> thinks.
> ------------------------------
>
> In public/style.css
> <https://github.com/ACM-IGDTUW/Open-Source-Challenge/pull/119#discussion_r410955080>
> :
>
> >  }
>
>  h2 {
> -  color: #313338;
> +  color: #ffffff;
>
> #ffffff can also be written as #fff, it means the same thing. Plus #fff
> is actually white and not black. But I don't see anything changing because
> of this?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ACM-IGDTUW/Open-Source-Challenge/pull/119#pullrequestreview-396044001>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANULTBFQAWGVEVRXUC4VOQDRNM2K7ANCNFSM4MLH47HA>
> .
>

lisa761 commented:
So, it'd be great if you changed the color back to what it was before, if it doesn't work :)

lisa761 commented on npm-debug.log:
```diff
@@ -0,0 +1,45 @@
+0 info it worked if it ends with ok
```
You seem to have added this extra `npm-debug.log` file in your commit. Make sure you remove this, as it would not be required. To see what I am talking about, you can click on that `Files Changed` button at the top of your PR. 
When you are adding files to a commit, you can run `git status` to see how many files have changed or been staged for the commit. If you changed multiple files, but only want to commit one, say `File1.md`. You can do that by running `add File.md` instead of `add .`. The `.` in the latter means that it adds _all_ the files that have changed.

sunitijain111 commented:
I guess there is a confusion. When the colour change didn't work, i made it
the original one. I guess it was FFFFFF or something.

On Wed, 22 Apr 2020, 11:42 pm Lisa Chandra, <notifications@github.com>
wrote:

> So, it'd be great if you changed the color back to what it was before, if
> it doesn't work :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ACM-IGDTUW/Open-Source-Challenge/pull/119#issuecomment-617943907>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANULTBFA6ZYQA2XW5D6H3T3RN4XSLANCNFSM4MLH47HA>
> .
>

lisa761 commented:
Did you check with your master? Or like I explained in the most recent review to see how you can check the changes you have made? Because it was `#313338` and _not_ `#ffffff`. Please make sure you read all the reviews correctly, and acknowledge them.

lisa761 commented:
Closing this since it's been inactive for a while. Feel free to reopen if you'd still like to get this merged :)
